docs/_docs/maintaining/merging-a-pull-request.md

```diff
@@ -30,12 +30,12 @@ The merge request is made up of three things:
 
 1. `@jekyllbot:` – this is the prefix our bot looks for when processing commands
 2. `merge` – the command
-3. `+dev` – the category to which the changes belong
+3. `+dev` – the category to which the changes belong. The `+` is optional.
```

Comment:
Really‽ TIL

Reply:
@pathawks I checked and we could also use :shipit: instead of merge 😄 

cf. https://github.com/parkr/auto-reply/blob/16e0487bc91d84b7e3af8efab31cf7b64880a8ac/chlog/merge_and_label.go#L27

Reply:
1. That is AWESOME!
2. From that line, it appears that the `+` is _not_ optional for the label to be respected. @parkr?

Reply:
@oe is right it works without the `+`, I just saw that in #6750

Reply:
Expect to see a lot more :shipit: in future PRs :)

Reply:
#6750 Was merged without being placed in the `docs` section. The `+` is _not_ optional.